utils/req-res-log-valiadtor.py

```diff
@@ -0,0 +1,270 @@
+#!/usr/bin/env python3
+import os
+import glob
+import json
+import jsonschema
+import subprocess
+import redis
+import time
+import argparse
+import multiprocessing
+import collections
+import io
+import signal
+import traceback
+from datetime import timedelta
+from functools import partial
+try:
+    from jsonschema import Draft201909Validator as schema_validator
+except ImportError:
+    from jsonschema import Draft7Validator as schema_validator
+
+"""
+The purpose of this file is to validate the reply_schema values of COMMAND DOCS.
+Basically, this is what it does:
+1. Goes over req-res files, generated by redis-servers, spawned by the testsuite (see logreqres.c)
+2. For each request-response pair, it validates the response against the request's reply_schema (obtained from COMMAND DOCS)
```

Comment:
let's also mention what it technically does (spins up a redis-server and uses redis-cli to get the COMMAND DOCS)